game.py

```diff
@@ -0,0 +1,127 @@
+from card import Card
+from color import Color
+from player import Player
+import constants
+import random
+
+
+class Game(Player, Card):
+    def __init__(self):
+        self.card_on_table = None
+        self.players = None
+        self.deck = None
+
+    def initialize_deck(self):
+        self.deck = [Card(num, color) for color in Color for num in range(1, 10)]
+        for i in range(constants.CHANGE_COLOR_AMOUNT):
+            self.deck.append(Card(is_change_color=True))
+        random.shuffle(self.deck)
+
+    @staticmethod
+    def get_number_of_players():
+        number_of_players = 0
+        while number_of_players not in range(constants.PLAYERS_LOWER_LIMIT, constants.PLAYERS_UPPER_LIMIT + 1):
+            number_of_players = int(input("Enter number of players (between 2 to 6): "))
+        return number_of_players
+
+    def initialize_players_data(self, number_of_players):
+        self.players = []
+        for i in range(number_of_players):
+            name = input(f"Insert player {i} name: ")
+            age = int(input(f"Insert player {i} age: "))
+            cards = [self.deck.pop() for j in range(constants.INITIAL_PLAYER_AMOUNT_OF_CARDS)]
+            player = Player(name, age, cards)
+            self.players.append(player)
+        self.players.sort(key=lambda x: x.age)
+
+    def put_initial_card_on_table(self):
+        self.card_on_table = self.deck.pop()
+        while self.card_on_table.change_color:
+            self.deck.append(self.card_on_table)
+            random.shuffle(self.deck)
+            self.card_on_table = self.deck.pop()
+
+    @staticmethod
+    def find_most_common_color(cards):
+        cards_colors = [card.color for card in cards if card.color]
+        if not len(cards_colors):
+            return None
+        most_common_color = max(set(cards_colors), key=cards_colors.count)
+        return most_common_color
+
+    def print_game_status(self):
+        print("Game status:")
+        print(f"There are {len(self.deck)} cards in the deck")
+        print(f"Card on table is {self.card_on_table}")
+        for player in self.players:
+            cards_text = [str(card) for card in player.cards]
+            print(f"{player.name} has {len(player.cards)} cards: {cards_text}")
+        print("")
+
+    def put_down_card_on_table(self, player, card):
+        self.card_on_table = card
+        player.cards.remove(card)
+
+    def play_matching_number_or_color_card(self, player):
+        for card in player.cards:
+            if card.color == self.card_on_table.color or card.num == self.card_on_table.num:
+                self.put_down_card_on_table(player, card)
+                print(f"{player.name} discarded {self.card_on_table} (player left with {len(player.cards)} cards) \n")
+                return True
+        return False
+
+    def play_change_color_card(self, player):
```

Comment:
From the name, I can understand each time you call this function you play a change color card, but it isn't what the function does, In addition it returns true or false, that I dont expect to have from the name of the function. 

suggestion: 
try_play_change_color_card